src/index.d.ts

```diff
@@ -87,6 +88,11 @@ export type IOSNativeProps = Readonly<BaseProps & {
    * instance, to show times in Pacific Standard Time, pass -7 * 60.
    */
   timeZoneOffsetInMinutes?: number,
+
+  /**
+   * The date picker text color.
+   */
+  textColor?: ColorValue,
```

Comment:
```suggestion
  textColor?: ?ColorValue,
```
let's please have this optional